physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -93,8 +93,8 @@
   type = integer
   intent = in
 [hwp_method_in]
-  standard_name = do_smoke_forecast
-  long_name = index for rrfs smoke forecast
+  standard_name = control_for_HWP_equation
+  long_name = control for HWP equation
   units = index
```

Comment:
units should be `1` according to https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst. This change should be here and everywhere else this variable is used in ccpp/physics and fv3atm.